File: WeatherGuide/Models/ApiModels/ChangeNameModel.cs
using System.ComponentModel.DataAnnotations;

namespace WeatherGuide.Models.ApiModels
{
    public class ChangeNameModel
    {
        [Required(ErrorMessage = "{0} is required")]
        [RegularExpression(@"^(([A-za-zА-ЩЬЮЯҐЄІЇа-щьюяґєії]+[\s]{1}[A-za-zА-ЩЬЮЯҐЄІЇа-щьюяґєії]+)|([A-Za-zА-ЩЬЮЯҐЄІЇа-щьюяґєії]+))$",
               ErrorMessage = "The Name field must contain alphabetical characters")]
        public string Name { get; set; }
        [Required(ErrorMessage = "{0} is required")]
        [RegularExpression(@"^(([A-za-zА-ЩЬЮЯҐЄІЇа-щьюяґєії]+[\s]{1}[A-za-zА-ЩЬЮЯҐЄІЇа-щьюяґєії]+)|([A-Za-zА-ЩЬЮЯҐЄІЇа-щьюяґєії]+))$",
            ErrorMessage = "The Surname field must contain alphabetical characters")]
        public string Surname { get; set; }
    }
}

File: android/android/WeatherGuide/app/src/main/java/com/dev/weatherguide/Model/ChangeNameModel.java
package com.dev.weatherguide.Model;

public class ChangeNameModel {
    String Name;
    String Surname;
    public ChangeNameModel(){

    }
   public ChangeNameModel(String name, String surname){
        Name = name;
        Surname = surname;
    }

    public String getName() {
        return Name;
    }

    public String getSurname() {
        return Surname;
    }

    public void setToken(String name) {
        Name = name;
    }

    public void setUser(String surname) {
        Surname = surname;
    }
}
